src/custom/pages/Profile/LockedGnoVesting/index.tsx

```diff
@@ -0,0 +1,166 @@
+  const unvested = allocated.subtract(vested)
+  const allocatedFormatted = formatSmartLocaleAware(allocated, AMOUNT_PRECISION) || '0'
+  const vestedFormatted = formatSmartLocaleAware(vested, AMOUNT_PRECISION) || '0'
+  const unvestedFormatted = formatSmartLocaleAware(unvested, AMOUNT_PRECISION) || '0'
+  const claimableFormatted = formatSmartLocaleAware(vested.subtract(claimed), AMOUNT_PRECISION) || '0'
+
+  const canClaim = !loadingBalances && unvested.greaterThan(0) && status === ClaimStatus.INITIAL
+  const isClaimPending = status === ClaimStatus.SUBMITTED
+
+  const { handleSetError, handleCloseError, ErrorModal } = useErrorModal()
+
+  const claimCallback = useClaimCallback({
+    openModal,
+    closeModal,
+    isFirstClaim: claimed.equalTo(0),
+  })
+
+  const handleClaim = useCallback(async () => {
+    handleCloseError()
+    if (!claimCallback) {
+      return
+    }
+
+    setStatus(ClaimStatus.ATTEMPTING)
+
+    claimCallback()
+      .then((tx) => {
+        setStatus(ClaimStatus.SUBMITTED)
+        return tx.wait()
+      })
+      .then(() => {
+        setStatus(ClaimStatus.CONFIRMED)
+
+        setTimeout(() => {
+          setStatus(ClaimStatus.INITIAL)
+        }, 5000)
+      })
+      .catch((error) => {
+        console.error('[Profile::LockedGnoVesting::index::claimCallback]::error', error)
+        setStatus(ClaimStatus.INITIAL)
+        handleSetError(error?.message)
+      })
+  }, [handleCloseError, handleSetError, claimCallback])
+
+  if (allocated.equalTo(0)) {
+    // don't render anything until we know that the user is actually eligible to claim
+    return null
+  }
+
+  return (
+    <>
+      <Card showLoader={loadingBalances || isClaimPending}>
+        <BalanceDisplay hAlign="left">
+          <img src={cowImage} alt="COW token" width="56" height="56" />
+          <span>
+            <i>COW vesting from locked GNO</i>
+            <b>
+              {allocatedFormatted} COW{' '}
+              <MouseoverTooltipContent
+                wrap
+                content={
+                  <VestingBreakdown>
+                    <span>
+                      <i>Unvested</i> <p>{unvestedFormatted} COW</p>
+                    </span>
+                    <span>
+                      <i>Vested</i> <p>{vestedFormatted} COW</p>
+                    </span>
+                  </VestingBreakdown>
+                }
+              >
+                <HelpCircle size={14} />
+              </MouseoverTooltipContent>
+            </b>
+          </span>
+        </BalanceDisplay>
+        <ConvertWrapper>
+          <BalanceDisplay titleSize={18} altColor={true}>
+            <i>
+              Vested{' '}
+              <MouseoverTooltipContent
+                wrap
+                content={
+                  <div>
+                    <p>
+                      <strong>COW vesting from the GNO lock</strong> is vested linearly over four years, starting on Fri
+                      Feb 11 2022 at 13:05:15 GMT.
```

Comment:
I suggest setting this timestamp as a Date object and formatting according to the locale like it was done for the airdrop https://github.com/cowprotocol/cowswap/blob/develop/src/custom/utils/time.ts#L13=